.github/workflows/ci.yml

```diff
@@ -139,22 +139,21 @@ jobs:
         .github/workflows/tests.sh
 
     - name: Upload coverage report
-      if: matrix.python-version == 3.5
-      uses: codecov/codecov-action@v1.0.5
+      if: matrix.python-version == 3.5 && github.repository == 'aiidateam/aiida-core'
+      uses: codecov/codecov-action@v1
```

Comment:
going back to older version?

Reply:
Yes. This is what they recommend. And they have updated the `v1` tag with the updates from the "newer" versions. I.e., `v1` is synonymous with the latest v1, no matter the minor or patch version.